### DISCUSSION TITLE
Bug Report: Talent "Mule" doesn't work with physicorium bar

### DISCUSSION THREAD

**Discussion by Instant2116** [FakeFishGames/Barotrauma#13954] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I was mining in an abyss and found out that physicorium stack up to regular 8 items per stack, while paralyxis stack up to 10 items per stack

### Reproduction steps

1. start single player campaign
2. give an assistant "Mule" talent
3. give said assistant 10 physicorium bars and 10 paralyxises 
4. Observe how physicorium bars stack up to 8, while paralyxises stack up to 10

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! This has been fixed in our private repo, fix coming up in a future update.
